package.json

```diff
@@ -39,6 +39,10 @@
     "chai": "^4.0.2",
     "enzyme": "^2.8.2",
     "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^15.0.2",
```

Comment:
👍  on adding these here. Should make setting up the development environment for this easier for future developers.